.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,13 @@
+name: Node.js
+on: [pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
Is it not possible to use the `apachepulsar/pulsar-build` image directly?

Reply:
You can probably do that with self-hosted runners.
https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners

Reply:
@massakam 
By default, it seems that only the virtual environments listed in the table on the following page can be used.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

@okuryu 
Thank you for your comment.
I will confirm self-hosted runners.

Reply:
I read the documents for self-hosted runners.
There are things to consider(security,server,etc), so we would like to consider self-hosted runners again in the future as needed.